From 079b95b2d16e8181e6099cb5a674e51d74332793-regenerated Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johasc@microsoft.com>
Date: Sun, 27 Sep 2026 19:57:45 +0200
Subject: [PATCH 3/3] win64/arm64: regenerate PE loader artifacts for TLS support

The preceding commit added ARM64 TLS-protection sections. Regenerate the
checked-in stub blob, relocation dump, and disassembly via `make -C
src/stub arm64-win64.pe.h` on Linux, using the current `upx-stubtools`.

Assisted-by: GPT-6 Sol
Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 src/stub/arm64-win64.pe.h            | 822 ++++++++++++++-------------
 src/stub/tmp/arm64-win64.pe.bin.dump |  55 +-
 2 files changed, 470 insertions(+), 407 deletions(-)

diff --git a/src/stub/arm64-win64.pe.h b/src/stub/arm64-win64.pe.h
index 22376791..6076058b 100644
--- a/src/stub/arm64-win64.pe.h
+++ b/src/stub/arm64-win64.pe.h
@@ -1,5 +1,5 @@
 /* arm64-win64.pe.h
-   created from arm64-win64.pe.bin, 6226 (0x1852) bytes
+   created from arm64-win64.pe.bin, 6968 (0x1b38) bytes
 
    This file is part of the UPX executable compressor.
 
@@ -32,399 +32,445 @@
 
 /* clang-format off */
 
-#define STUB_ARM64_WIN64_PE_SIZE    6226
-#define STUB_ARM64_WIN64_PE_ADLER32 0x9b782a26
-#define STUB_ARM64_WIN64_PE_CRC32   0x8db0c44c
+#define STUB_ARM64_WIN64_PE_SIZE    6968
+#define STUB_ARM64_WIN64_PE_ADLER32 0xe6f1d38f
+#define STUB_ARM64_WIN64_PE_CRC32   0x026b2bab
 
-unsigned char stub_arm64_win64_pe[6226] = {
+unsigned char stub_arm64_win64_pe[6968] = {
 /* 0x0000 */ 127, 69, 76, 70,  2,  1,  1,  0,  0,  0,  0,  0,  0,  0,  0,  0,
 /* 0x0010 */   1,  0,183,  0,  1,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
-/* 0x0020 */   0,  0,  0,  0,  0,  0,  0,  0, 96, 13,  0,  0,  0,  0,  0,  0,
+/* 0x0020 */   0,  0,  0,  0,  0,  0,  0,  0,160, 14,  0,  0,  0,  0,  0,  0,
 /* 0x0030 */   0,  0,  0,  0, 64,  0,  0,  0,  0,  0, 64,  0,  0,  0,  0,  0,
-/* 0x0040 */ 253,123,190,169,243, 83,  1,169,145,  0,  0, 16, 48,  2,128,185,
-/* 0x0050 */  52,  2, 16,139,  2,  0,  0, 20,  0,  0,  0,  0,145,  0,  0, 16,
-/* 0x0060 */  48,  2, 64,185,  9,126, 64,147,  2,  0,  0, 20,  0,  0,  0,  0,
-/* 0x0070 */ 147,  2,  9,139,224,  3, 20,170,145,  0,  0, 16, 48,  2, 64,185,
-/* 0x0080 */   1,126, 64,147,  2,  0,  0, 20,  0,  0,  0,  0,226,  3, 19,170,
-/* 0x0090 */ 255, 67,  0,209,227,  3,  0,145,228,  3, 31, 42,  0,  0,  0,148,
-/* 0x00a0 */   0,  0,  0,148,  0,  0,  0,148,255, 67,  0,145,145,  0,  0, 16,
-/* 0x00b0 */  48,  2, 64,185,  9,126, 64,147,  2,  0,  0, 20,  0,  0,  0,  0,
-/* 0x00c0 */ 117,  2,  9,139,169,  2, 64,185,201,  4,  0, 52,170,  6, 64,185,
-/* 0x00d0 */ 145,  0,  0, 16, 48,  2, 64,185, 11,126, 64,147,  2,  0,  0, 20,
-/* 0x00e0 */   0,  0,  0,  0, 96,  2,  9,139,  0,  0, 11,139,118,  2, 10,139,
-/* 0x00f0 */ 181, 34,  0,145,145,  0,  0, 16, 48,  2,128,185, 44,  2, 16,139,
-/* 0x0100 */   2,  0,  0, 20,  0,  0,  0,  0,140,  1, 64,249,128,  1, 63,214,
-/* 0x0110 */ 247,  3,  0,170,169, 22, 64, 56,105,253,255, 52, 63,  5,  0,113,
-/* 0x0120 */ 161,  0,  0, 84,225,  3, 21,170,170, 22, 64, 56,234,255,255, 53,
-/* 0x0130 */   2,  0,  0, 20,161, 38, 64,120,224,  3, 23,170,145,  0,  0, 16,
-/* 0x0140 */  48,  2,128,185, 44,  2, 16,139,  2,  0,  0, 20,  0,  0,  0,  0,
-/* 0x0150 */ 140,  1, 64,249,128,  1, 63,214,192,134,  0,248,238,255,255, 23,
-/* 0x0160 */ 145,  0,  0, 16, 48,  2, 64,185,  9,126, 64,147,  2,  0,  0, 20,
-/* 0x0170 */   0,  0,  0,  0,117,  2,  9,139,118, 18,  0,209,169, 22, 64, 56,
-/* 0x0180 */  73,  2,  0, 52, 63,189,  3,113,232,  0,  0, 84,214,  2,  9,139,
-/* 0x0190 */ 202,  2, 64,249, 74, 13,192,218, 74,  1, 19,139,202,  2,  0,249,
-/* 0x01a0 */ 247,255,255, 23, 41, 13,  0, 18, 41, 61, 16, 83,170, 22, 64, 56,
-/* 0x01b0 */ 171, 22, 64, 56, 41,  1, 10, 42, 41, 33, 11, 42,137,254,255, 53,
-/* 0x01c0 */ 169, 70, 64,184,242,255,255, 23,  0,  0,128,146,225,  3, 19,170,
-/* 0x01d0 */ 145,  0,  0, 16, 48,  2, 64,185,  2,126, 64,147,  2,  0,  0, 20,
-/* 0x01e0 */   0,  0,  0,  0,145,  0,  0, 16, 48,  2,128,185, 41,  2, 16,139,
-/* 0x01f0 */   2,  0,  0, 20,  0,  0,  0,  0, 41,  1, 64,249, 32,  1, 63,214,
-/* 0x0200 */ 145,  0,  0, 16, 48,  2,128,185, 52,  2, 16,139,  2,  0,  0, 20,
-/* 0x0210 */   0,  0,  0,  0,234,  3,132, 82,106,160,186,114,138,  2,  0,185,
-/* 0x0220 */   0,  0,128,146,225,  3, 20,170,130,  0,128,210,145,  0,  0, 16,
-/* 0x0230 */  48,  2,128,185, 41,  2, 16,139,  2,  0,  0, 20,  0,  0,  0,  0,
-/* 0x0240 */  41,  1, 64,249, 32,  1, 63,214,224,  3, 19,170, 33,  0,128, 82,
-/* 0x0250 */ 226,  3, 31,170,  0,  0,  0,148,243, 83, 65,169,253,123,194,168,
-/* 0x0260 */   0,  0,  0, 20,  1, 64, 33,139,225, 11,190,169,227,123,  1,169,
-/* 0x0270 */   5,  0,128, 18,  4,  0,176, 82, 26,  0,  0, 20,228,123, 65,169,
-/* 0x0280 */ 225, 15,194,168,  0,  0,  1,203, 66,  0,  3,203,130,  0,  0,185,
-/* 0x0290 */ 229,  3,  0,170,224,  3,  3,170, 97,  0,  2,139,224,  3,  5,170,
-/* 0x02a0 */ 192,  3, 95,214,132,  0,  4, 43, 68,  0,  0, 52,192,  3, 95,214,
-/* 0x02b0 */   4, 68, 64,184,132,  0,  4, 58,192,  3, 95,214, 33,  0,128, 82,
-/* 0x02c0 */ 240,  3, 30,170,248,255,255,151, 33,  0,  1, 58,246,255,255,151,
-/* 0x02d0 */ 163,255,255, 84,  0,  2, 31,214,  3, 20, 64, 56, 67, 20,  0, 56,
-/* 0x02e0 */ 241,255,255,151,162,255,255, 84,245,255,255,151, 35, 12,  0,113,
-/* 0x02f0 */   1,  0,128, 82,163,  0,  0, 84,  5, 20, 64, 56,165, 32,  3, 42,
-/* 0x0300 */ 229,  3, 37, 42,197,251,255, 52,231,255,255,151, 33,  0,  1, 58,
-/* 0x0310 */ 229,255,255,151, 33,  0,  1, 58, 97,  0,  0, 84,232,255,255,151,
-/* 0x0320 */  33,  8,  0, 17,191,  0, 52, 49, 33, 36,129, 26, 67,200,101, 56,
-/* 0x0330 */  33,  4,  0,113, 67, 20,  0, 56,162,255,255, 84,233,255,255, 23,
-/* 0x0340 */ 226, 15,190,169,254, 11,  0,249,  7, 64, 33,139,  5,  0,128, 18,
-/* 0x0350 */   4,  0,176, 82, 19,  0,  0, 20,227, 19,193,168,  0,  0,  7,203,
-/* 0x0360 */  66,  0,  3,203,130,  0,  0,185,229,  3,  0,170,224,  3,  3,170,
-/* 0x0370 */  97,  0,  2,139,224,  3,  5,170,254,  7, 65,248,192,  3, 95,214,
-/* 0x0380 */ 132,  0,  4, 43, 68,  0,  0, 52,192,  3, 95,214,  4, 68, 64,184,
-/* 0x0390 */ 132,  0,  4, 58,192,  3, 95,214,  3, 20, 64, 56, 67, 20,  0, 56,
-/* 0x03a0 */ 248,255,255,151,162,255,255, 84, 33,  0,128, 82,  4,  0,  0, 20,
-/* 0x03b0 */  33,  4,  0, 81,243,255,255,151, 33,  0,  1, 58,241,255,255,151,
-/* 0x03c0 */  33,  0,  1, 58,239,255,255,151, 67,255,255, 84, 35, 12,  0,113,
-/* 0x03d0 */   1,  0,128, 82, 35,  1,  0, 84,  5, 20, 64, 56,165, 32,  3, 42,
-/* 0x03e0 */ 229,  3, 37, 42,165,251,255, 52,163,  0,  1, 83,165,124,  1, 19,
-/* 0x03f0 */  99,  0,  3, 43,  2,  0,  0, 20,226,255,255,151, 33,  0,  1, 58,
-/* 0x0400 */ 224,255,255,151, 33,  0,  1, 58,225,  0,  0, 84, 33,  0,128, 82,
-/* 0x0410 */ 220,255,255,151, 33,  0,  1, 58,218,255,255,151,163,255,255, 84,
-/* 0x0420 */  33,  8,  0, 17, 33,  4,  0, 17,191,  0, 20, 49, 33, 36,129, 26,
-/* 0x0430 */  67,  0, 64, 57, 67,200,101, 56, 67, 20,  0, 56, 33,  4,  0,113,
-/* 0x0440 */ 161,255,255, 84,215,255,255, 23,254, 15, 31,248,226, 15,191,169,
-/* 0x0450 */   7, 64, 33,139,  5,  0,128, 18,  4,  0,176, 82, 19,  0,  0, 20,
-/* 0x0460 */ 227, 19,193,168,  0,  0,  7,203, 66,  0,  3,203,130,  0,  0,185,
-/* 0x0470 */ 229,  3,  0,170,224,  3,  3,170, 97,  0,  2,139,224,  3,  5,170,
-/* 0x0480 */ 254,  7, 65,248,192,  3, 95,214,132,  0,  4, 43, 68,  0,  0, 52,
-/* 0x0490 */ 192,  3, 95,214,  4, 68, 64,184,132,  0,  4, 58,192,  3, 95,214,
-/* 0x04a0 */   3, 20, 64, 56, 67, 20,  0, 56,248,255,255,151,162,255,255, 84,
-/* 0x04b0 */  33,  0,128, 82,  4,  0,  0, 20, 33,  4,  0, 81,243,255,255,151,
-/* 0x04c0 */  33,  0,  1, 26,241,255,255,151, 33,  0,  1, 26,239,255,255,151,
-/* 0x04d0 */  67,255,255, 84, 35, 12,  0,113,  1,  0,128, 82, 35,  1,  0, 84,
-/* 0x04e0 */   5, 20, 64, 56,165, 32,  3, 42,229,  3, 37, 42,165,251,255, 52,
-/* 0x04f0 */ 191,  0,  0,114,165,124,  1, 19,161,  1,  0, 84,  3,  0,  0, 20,
-/* 0x0500 */ 226,255,255,151, 66,  1,  0, 84, 33,  0,128, 82,223,255,255,151,
-/* 0x0510 */ 226,  0,  0, 84,221,255,255,151, 33,  0,  1, 26,219,255,255,151,
-/* 0x0520 */ 163,255,255, 84, 33, 16,  0, 17,  4,  0,  0, 20,215,255,255,151,
-/* 0x0530 */  33,  0,  1, 26, 33,  8,  0, 17,191,  0, 20, 49, 33, 36,129, 26,
-/* 0x0540 */  67, 64, 33,139, 99,240, 95, 56, 67,200,101, 56, 67, 20,  0, 56,
-/* 0x0550 */  33,  4,  0,113,161,255,255, 84,212,255,255, 23,192,  3, 95,214,
-/* 0x0560 */ 253,123,189,169,243, 83,  1,169,245, 91,  2,169,243,  3,  0,170,
-/* 0x0570 */ 244,  3,  1,170,245,  3,  2,170,145,  0,  0, 16, 48,  2,128,185,
-/* 0x0580 */  54,  2, 16,139,  2,  0,  0, 20,  0,  0,  0,  0,201,134, 64,248,
-/* 0x0590 */ 201,  0,  0,180,224,  3, 19,170,225,  3, 20, 42,226,  3, 21,170,
-/* 0x05a0 */  32,  1, 63,214,250,255,255, 23,245, 91, 66,169,243, 83, 65,169,
-/* 0x05b0 */ 253,123,195,168,192,  3, 95,214,  0,  0,  0,  0,  0,  0,  0,  0,
-/* 0x05c0 */  85, 80, 88, 33,161,216,208,213,  0,  0,  0,  0,  0,  0,  0,  0,
-/* 0x05d0 */   0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 45,
-/* 0x05e0 */ 102,105,108,101, 32,102,111,114,109, 97,116, 32,101,108,102, 54,
-/* 0x05f0 */  52, 45,108,105,116,116,108,101, 97, 97,114, 99,104, 54, 52, 10,
-/* 0x0600 */  10, 83,101, 99,116,105,111,110,115, 58, 10, 73,100,120, 32, 78,
-/* 0x0610 */  97,109,101, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 83,105,122,
-/* 0x0620 */ 101, 32, 32, 32, 32, 32, 32, 86, 77, 65, 32, 32, 32, 32, 32, 32,
-/* 0x0630 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 76, 77, 65, 32, 32, 32, 32,
-/* 0x0640 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 70,105,108,101, 32,
-/* 0x0650 */ 111,102,102, 32, 32, 65,108,103,110, 32, 32, 70,108, 97,103,115,
-/* 0x0660 */  10, 32, 32, 48, 32, 83, 84, 65, 82, 84, 32, 32, 32, 32, 32, 32,
-/* 0x0670 */  32, 32, 32, 48, 32, 32, 48, 32, 32, 48, 32, 32, 48, 52, 48, 32,
-/* 0x0680 */  32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10,
-/* 0x0690 */  32, 32, 49, 32, 80, 69, 77, 65, 73, 78, 48, 49, 32, 32, 32, 32,
-/* 0x06a0 */  32, 32, 48, 51, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48, 52, 48,
-/* 0x06b0 */  32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83,
-/* 0x06c0 */  10, 32, 32, 50, 32, 80, 69, 77, 65, 73, 78, 48, 50, 32, 32, 32,
-/* 0x06d0 */  32, 32, 32, 48, 50, 56, 32, 32, 48, 32, 32, 48, 32, 32, 48, 55,
-/* 0x06e0 */  52, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84,
-/* 0x06f0 */  83, 10, 32, 32, 51, 32, 80, 69, 67, 65, 76, 76, 50, 66, 32, 32,
-/* 0x0700 */  32, 32, 32, 32, 48, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48, 57,
-/* 0x0710 */  99, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84,
-/* 0x0720 */  83, 10, 32, 32, 52, 32, 80, 69, 67, 65, 76, 76, 50, 68, 32, 32,
-/* 0x0730 */  32, 32, 32, 32, 48, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48, 97,
-/* 0x0740 */  48, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84,
-/* 0x0750 */  83, 10, 32, 32, 53, 32, 80, 69, 67, 65, 76, 76, 50, 69, 32, 32,
-/* 0x0760 */  32, 32, 32, 32, 48, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48, 97,
-/* 0x0770 */  52, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84,
-/* 0x0780 */  83, 10, 32, 32, 54, 32, 80, 69, 77, 65, 73, 78, 49, 48, 32, 32,
-/* 0x0790 */  32, 32, 32, 32, 48, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48, 97,
-/* 0x07a0 */  56, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84,
-/* 0x07b0 */  83, 10, 32, 32, 55, 32, 80, 69, 73, 77, 80, 79, 82, 84, 32, 32,
-/* 0x07c0 */  32, 32, 32, 32, 48, 98, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48,
-/* 0x07d0 */  97, 99, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78,
-/* 0x07e0 */  84, 83, 10, 32, 32, 56, 32, 80, 69, 82, 69, 76, 79, 67, 49, 32,
-/* 0x07f0 */  32, 32, 32, 32, 32, 48, 54, 56, 32, 32, 48, 32, 32, 48, 32, 32,
-/* 0x0800 */  48, 49, 54, 48, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84,
-/* 0x0810 */  69, 78, 84, 83, 10, 32, 32, 57, 32, 80, 69, 67, 65, 67, 72, 69,
-/* 0x0820 */  32, 32, 32, 32, 32, 32, 32, 48, 51, 56, 32, 32, 48, 32, 32, 48,
-/* 0x0830 */  32, 32, 48, 49, 99, 56, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79,
-/* 0x0840 */  78, 84, 69, 78, 84, 83, 10, 32, 49, 48, 32, 80, 69, 84, 76, 83,
-/* 0x0850 */  67, 32, 32, 32, 32, 32, 32, 32, 32, 48, 53, 56, 32, 32, 48, 32,
-/* 0x0860 */  32, 48, 32, 32, 48, 50, 48, 48, 32, 32, 50, 42, 42, 48, 32, 32,
-/* 0x0870 */  67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 49, 49, 32, 80, 69, 68,
-/* 0x0880 */  79, 74, 85, 77, 80, 32, 32, 32, 32, 32, 32, 48, 99, 32, 32, 48,
-/* 0x0890 */  32, 32, 48, 32, 32, 48, 50, 53, 56, 32, 32, 50, 42, 42, 48, 32,
-/* 0x08a0 */  32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 49, 50, 32, 78, 82,
-/* 0x08b0 */  86, 95, 72, 69, 65, 68, 32, 32, 32, 32, 32, 32, 48, 32, 32, 48,
-/* 0x08c0 */  32, 32, 48, 32, 32, 48, 50, 54, 52, 32, 32, 50, 42, 42, 48, 32,
-/* 0x08d0 */  32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 49, 51, 32, 78, 82,
-/* 0x08e0 */  86, 50, 66, 32, 32, 32, 32, 32, 32, 32, 32, 32, 48,100, 99, 32,
-/* 0x08f0 */  32, 48, 32, 32, 48, 32, 32, 48, 50, 54, 52, 32, 32, 50, 42, 42,
-/* 0x0900 */  48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 49, 52, 32,
-/* 0x0910 */  78, 82, 86, 50, 68, 32, 32, 32, 32, 32, 32, 32, 32, 32, 48, 49,
-/* 0x0920 */  48, 56, 32, 32, 48, 32, 32, 48, 32, 32, 48, 51, 52, 48, 32, 32,
-/* 0x0930 */  50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32,
-/* 0x0940 */  49, 53, 32, 78, 82, 86, 50, 69, 32, 32, 32, 32, 32, 32, 32, 32,
-/* 0x0950 */  32, 48, 49, 49, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48, 52, 52,
-/* 0x0960 */  56, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84,
-/* 0x0970 */  83, 10, 32, 49, 54, 32, 80, 69, 84, 76, 83, 67, 50, 32, 32, 32,
-/* 0x0980 */  32, 32, 32, 32, 48, 53, 99, 32, 32, 48, 32, 32, 48, 32, 32, 48,
-/* 0x0990 */  53, 53, 99, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69,
-/* 0x09a0 */  78, 84, 83, 10, 32, 49, 55, 32, 80, 69, 70, 73, 76, 84, 83, 89,
-/* 0x09b0 */  77, 32, 32, 32, 32, 32, 48, 56, 32, 32, 48, 32, 32, 48, 32, 32,
-/* 0x09c0 */  48, 53, 98, 56, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84,
-/* 0x09d0 */  69, 78, 84, 83, 10, 32, 49, 56, 32, 85, 80, 88, 49, 72, 69, 65,
-/* 0x09e0 */  68, 32, 32, 32, 32, 32, 32, 48, 50, 48, 32, 32, 48, 32, 32, 48,
-/* 0x09f0 */  32, 32, 48, 53, 99, 48, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79,
-/* 0x0a00 */  78, 84, 69, 78, 84, 83, 10, 83, 89, 77, 66, 79, 76, 32, 84, 65,
-/* 0x0a10 */  66, 76, 69, 58, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0a20 */  48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69,
-/* 0x0a30 */  84, 76, 83, 67, 50, 32, 48, 32, 80, 69, 84, 76, 83, 67, 50, 10,
-/* 0x0a40 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0a50 */  32,108, 32, 32, 32, 32,100, 32, 32, 83, 84, 65, 82, 84, 32, 48,
-/* 0x0a60 */  32, 83, 84, 65, 82, 84, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0a70 */  48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32,
-/* 0x0a80 */  80, 69, 77, 65, 73, 78, 48, 49, 32, 48, 32, 80, 69, 77, 65, 73,
-/* 0x0a90 */  78, 48, 49, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0aa0 */  48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 77,
-/* 0x0ab0 */  65, 73, 78, 48, 50, 32, 48, 32, 80, 69, 77, 65, 73, 78, 48, 50,
-/* 0x0ac0 */  10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0ad0 */  48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 67, 65, 76, 76,
-/* 0x0ae0 */  50, 66, 32, 48, 32, 80, 69, 67, 65, 76, 76, 50, 66, 10, 48, 48,
-/* 0x0af0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108,
-/* 0x0b00 */  32, 32, 32, 32,100, 32, 32, 80, 69, 67, 65, 76, 76, 50, 68, 32,
-/* 0x0b10 */  48, 32, 80, 69, 67, 65, 76, 76, 50, 68, 10, 48, 48, 48, 48, 48,
-/* 0x0b20 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32,
-/* 0x0b30 */  32,100, 32, 32, 80, 69, 67, 65, 76, 76, 50, 69, 32, 48, 32, 80,
-/* 0x0b40 */  69, 67, 65, 76, 76, 50, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0b50 */  48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32,
-/* 0x0b60 */  32, 80, 69, 77, 65, 73, 78, 49, 48, 32, 48, 32, 80, 69, 77, 65,
-/* 0x0b70 */  73, 78, 49, 48, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0b80 */  48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69,
-/* 0x0b90 */  73, 77, 80, 79, 82, 84, 32, 48, 32, 80, 69, 73, 77, 80, 79, 82,
-/* 0x0ba0 */  84, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0bb0 */  48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 82, 69, 76,
-/* 0x0bc0 */  79, 67, 49, 32, 48, 32, 80, 69, 82, 69, 76, 79, 67, 49, 10, 48,
-/* 0x0bd0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,
-/* 0x0be0 */ 108, 32, 32, 32, 32,100, 32, 32, 80, 69, 67, 65, 67, 72, 69, 32,
-/* 0x0bf0 */  48, 32, 80, 69, 67, 65, 67, 72, 69, 10, 48, 48, 48, 48, 48, 48,
-/* 0x0c00 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,
-/* 0x0c10 */ 100, 32, 32, 80, 69, 84, 76, 83, 67, 32, 48, 32, 80, 69, 84, 76,
-/* 0x0c20 */  83, 67, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0c30 */  48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 68, 79,
-/* 0x0c40 */  74, 85, 77, 80, 32, 48, 32, 80, 69, 68, 79, 74, 85, 77, 80, 10,
-/* 0x0c50 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0c60 */  32,108, 32, 32, 32, 32,100, 32, 32, 78, 82, 86, 95, 72, 69, 65,
-/* 0x0c70 */  68, 32, 48, 32, 78, 82, 86, 95, 72, 69, 65, 68, 10, 48, 48, 48,
-/* 0x0c80 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32,
-/* 0x0c90 */  32, 32, 32,100, 32, 32, 78, 82, 86, 50, 66, 32, 48, 32, 78, 82,
-/* 0x0ca0 */  86, 50, 66, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0cb0 */  48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 78, 82, 86,
-/* 0x0cc0 */  50, 68, 32, 48, 32, 78, 82, 86, 50, 68, 10, 48, 48, 48, 48, 48,
-/* 0x0cd0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32,
-/* 0x0ce0 */  32,100, 32, 32, 78, 82, 86, 50, 69, 32, 48, 32, 78, 82, 86, 50,
-/* 0x0cf0 */  69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0d00 */  48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 70, 73, 76,
-/* 0x0d10 */  84, 83, 89, 77, 32, 48, 32, 80, 69, 70, 73, 76, 84, 83, 89, 77,
-/* 0x0d20 */  10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0d30 */  48, 32,108, 32, 32, 32, 32,100, 32, 32, 85, 80, 88, 49, 72, 69,
-/* 0x0d40 */  65, 68, 32, 48, 32, 85, 80, 88, 49, 72, 69, 65, 68, 10, 48, 48,
-/* 0x0d50 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32,
-/* 0x0d60 */  32, 32, 32, 32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32,115,
-/* 0x0d70 */ 116, 97,114,116, 95,111,102, 95, 99,111,109,112,114,101,115,115,
-/* 0x0d80 */ 101,100, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0d90 */  48, 48, 48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78, 68,
-/* 0x0da0 */  42, 32, 48, 32,115,116, 97,114,116, 95,111,102, 95,117,110, 99,
-/* 0x0db0 */ 111,109,112,114,101,115,115,101,100, 10, 48, 48, 48, 48, 48, 48,
-/* 0x0dc0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32,
-/* 0x0dd0 */  32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32, 99,111,109,112, 95,
-/* 0x0de0 */ 108,101,110, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0df0 */  48, 48, 48, 48, 32,103, 32, 32, 32, 32, 32, 70, 32, 78, 82, 86,
-/* 0x0e00 */  50, 66, 32, 48,100, 99, 32,117, 99,108, 95,110,114,118, 50, 98,
-/* 0x0e10 */  95,100,101, 99,111,109,112,114,101,115,115, 95, 51, 50, 10, 48,
-/* 0x0e20 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,
-/* 0x0e30 */ 103, 32, 32, 32, 32, 32, 70, 32, 78, 82, 86, 50, 68, 32, 48, 49,
-/* 0x0e40 */  48, 56, 32,117, 99,108, 95,110,114,118, 50,100, 95,100,101, 99,
-/* 0x0e50 */ 111,109,112,114,101,115,115, 95, 51, 50, 10, 48, 48, 48, 48, 48,
-/* 0x0e60 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,103, 32, 32, 32,
-/* 0x0e70 */  32, 32, 70, 32, 78, 82, 86, 50, 69, 32, 48, 49, 49, 52, 32,117,
-/* 0x0e80 */  99,108, 95,110,114,118, 50,101, 95,100,101, 99,111,109,112,114,
-/* 0x0e90 */ 101,115,115, 95, 51, 50, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x0ea0 */  48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-/* 0x0eb0 */  42, 85, 78, 68, 42, 32, 48, 32, 99,111,109,112,114,101,115,115,
-/* 0x0ec0 */ 101,100, 95,105,109,112,111,114,116,115, 10, 48, 48, 48, 48, 48,
-/* 0x0ed0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32,
-/* 0x0ee0 */  32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32,115,116, 97,114,
-/* 0x0ef0 */ 116, 95,111,102, 95,105,109,112,111,114,116,115, 10, 48, 48, 48,
-/* 0x0f00 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32,
-/* 0x0f10 */  32, 32, 32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32, 76,111,
-/* 0x0f20 */  97,100, 76,105, 98,114, 97,114,121, 65, 10, 48, 48, 48, 48, 48,
-/* 0x0f30 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32,
-/* 0x0f40 */  32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32, 71,101,116, 80,
-/* 0x0f50 */ 114,111, 99, 65,100,100,114,101,115,115, 10, 48, 48, 48, 48, 48,
-/* 0x0f60 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32,
-/* 0x0f70 */  32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32,115,116, 97,114,
-/* 0x0f80 */ 116, 95,111,102, 95,114,101,108,111, 99,115, 10, 48, 48, 48, 48,
-/* 0x0f90 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32,
-/* 0x0fa0 */  32, 32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32,115,105,122,
-/* 0x0fb0 */ 101,111,102, 95,105,109, 97,103,101, 10, 48, 48, 48, 48, 48, 48,
-/* 0x0fc0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32,
-/* 0x0fd0 */  32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32, 70,108,117,115,104,
-/* 0x0fe0 */  73,110,115,116,114,117, 99,116,105,111,110, 67, 97, 99,104,101,
-/* 0x0ff0 */  10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1000 */  48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78, 68, 42, 32,
-/* 0x1010 */  48, 32,111,114,105,103,105,110, 97,108, 95,101,110,116,114,121,
-/* 0x1020 */  10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1030 */  48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78, 68, 42, 32,
-/* 0x1040 */  48, 32,116,108,115, 95, 99, 97,108,108, 98, 97, 99,107,115, 95,
-/* 0x1050 */ 112,116,114, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1060 */  48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78,
-/* 0x1070 */  68, 42, 32, 48, 32,102,105,108,116,101,114, 95,108,101,110,103,
-/* 0x1080 */ 116,104, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1090 */  48, 48, 48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78, 68,
-/* 0x10a0 */  42, 32, 48, 32,102,105,108,116,101,114, 95, 99,116,111, 10, 10,
-/* 0x10b0 */  82, 69, 76, 79, 67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82,
-/* 0x10c0 */  68, 83, 32, 70, 79, 82, 32, 91, 80, 69, 77, 65, 73, 78, 48, 49,
-/* 0x10d0 */  93, 58, 10, 79, 70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32,
-/* 0x10e0 */  32, 32, 32, 32, 84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32,
-/* 0x10f0 */  32, 32, 32, 32, 32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48,
-/* 0x1100 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 49, 56, 32, 82, 95, 65,
-/* 0x1110 */  65, 82, 67, 72, 54, 52, 95, 80, 82, 69, 76, 51, 50, 32, 32,115,
-/* 0x1120 */ 116, 97,114,116, 95,111,102, 95, 99,111,109,112,114,101,115,115,
-/* 0x1130 */ 101,100, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1140 */  48, 50, 99, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 65, 66,
-/* 0x1150 */  83, 51, 50, 32, 32, 32,115,116, 97,114,116, 95,111,102, 95,117,
-/* 0x1160 */ 110, 99,111,109,112,114,101,115,115,101,100, 10, 10, 82, 69, 76,
-/* 0x1170 */  79, 67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32,
-/* 0x1180 */  70, 79, 82, 32, 91, 80, 69, 77, 65, 73, 78, 48, 50, 93, 58, 10,
-/* 0x1190 */  79, 70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-/* 0x11a0 */  32, 84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-/* 0x11b0 */  32, 32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48,
-/* 0x11c0 */  48, 48, 48, 48, 48, 48, 48, 49, 52, 32, 82, 95, 65, 65, 82, 67,
-/* 0x11d0 */  72, 54, 52, 95, 65, 66, 83, 51, 50, 32, 32, 32, 99,111,109,112,
-/* 0x11e0 */  95,108,101,110, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79, 78,
-/* 0x11f0 */  32, 82, 69, 67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80, 69,
-/* 0x1200 */  67, 65, 76, 76, 50, 66, 93, 58, 10, 79, 70, 70, 83, 69, 84, 32,
-/* 0x1210 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32, 32,
-/* 0x1220 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76, 85,
-/* 0x1230 */  69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1240 */  48, 48, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 67, 65, 76,
-/* 0x1250 */  76, 50, 54, 32, 32,117, 99,108, 95,110,114,118, 50, 98, 95,100,
-/* 0x1260 */ 101, 99,111,109,112,114,101,115,115, 95, 51, 50, 10, 10, 82, 69,
+/* 0x0040 */  63,  4,  0,113, 64,  0,  0, 84,  0,  0,  0, 20,253,123,190,169,
+/* 0x0050 */ 243, 83,  1,169,145,  0,  0, 16, 48,  2,128,185, 52,  2, 16,139,
+/* 0x0060 */   2,  0,  0, 20,  0,  0,  0,  0,145,  0,  0, 16, 48,  2, 64,185,
+/* 0x0070 */   9,126, 64,147,  2,  0,  0, 20,  0,  0,  0,  0,147,  2,  9,139,
+/* 0x0080 */ 145,  0,  0, 16, 48,  2, 64,185,  9,126, 64,147,  2,  0,  0, 20,
+/* 0x0090 */   0,  0,  0,  0,105,  2,  9,139, 42,  1, 64,185,233, 43,191,169,
+/* 0x00a0 */ 145,  0,  0, 16, 48,  2, 64,185, 11,126, 64,147,  2,  0,  0, 20,
+/* 0x00b0 */   0,  0,  0,  0, 43,  1,  0,185,224,  3, 20,170,145,  0,  0, 16,
+/* 0x00c0 */  48,  2, 64,185,  1,126, 64,147,  2,  0,  0, 20,  0,  0,  0,  0,
+/* 0x00d0 */ 226,  3, 19,170,255, 67,  0,209,227,  3,  0,145,228,  3, 31, 42,
+/* 0x00e0 */   0,  0,  0,148,  0,  0,  0,148,  0,  0,  0,148,255, 67,  0,145,
+/* 0x00f0 */ 233, 43,193,168, 42,  1,  0,185,145,  0,  0, 16, 48,  2, 64,185,
+/* 0x0100 */   9,126, 64,147,  2,  0,  0, 20,  0,  0,  0,  0,117,  2,  9,139,
+/* 0x0110 */ 169,  2, 64,185,201,  4,  0, 52,170,  6, 64,185,145,  0,  0, 16,
+/* 0x0120 */  48,  2, 64,185, 11,126, 64,147,  2,  0,  0, 20,  0,  0,  0,  0,
+/* 0x0130 */  96,  2,  9,139,  0,  0, 11,139,118,  2, 10,139,181, 34,  0,145,
+/* 0x0140 */ 145,  0,  0, 16, 48,  2,128,185, 44,  2, 16,139,  2,  0,  0, 20,
+/* 0x0150 */   0,  0,  0,  0,140,  1, 64,249,128,  1, 63,214,247,  3,  0,170,
+/* 0x0160 */ 169, 22, 64, 56,105,253,255, 52, 63,  5,  0,113,161,  0,  0, 84,
+/* 0x0170 */ 225,  3, 21,170,170, 22, 64, 56,234,255,255, 53,  2,  0,  0, 20,
+/* 0x0180 */ 161, 38, 64,120,224,  3, 23,170,145,  0,  0, 16, 48,  2,128,185,
+/* 0x0190 */  44,  2, 16,139,  2,  0,  0, 20,  0,  0,  0,  0,140,  1, 64,249,
+/* 0x01a0 */ 128,  1, 63,214,192,134,  0,248,238,255,255, 23,145,  0,  0, 16,
+/* 0x01b0 */  48,  2, 64,185,  9,126, 64,147,  2,  0,  0, 20,  0,  0,  0,  0,
+/* 0x01c0 */ 117,  2,  9,139,118, 18,  0,209,169, 22, 64, 56, 73,  2,  0, 52,
+/* 0x01d0 */  63,189,  3,113,232,  0,  0, 84,214,  2,  9,139,202,  2, 64,249,
+/* 0x01e0 */  74, 13,192,218, 74,  1, 19,139,202,  2,  0,249,247,255,255, 23,
+/* 0x01f0 */  41, 13,  0, 18, 41, 61, 16, 83,170, 22, 64, 56,171, 22, 64, 56,
+/* 0x0200 */  41,  1, 10, 42, 41, 33, 11, 42,137,254,255, 53,169, 70, 64,184,
+/* 0x0210 */ 242,255,255, 23,  0,  0,128,146,225,  3, 19,170,145,  0,  0, 16,
+/* 0x0220 */  48,  2, 64,185,  2,126, 64,147,  2,  0,  0, 20,  0,  0,  0,  0,
+/* 0x0230 */ 145,  0,  0, 16, 48,  2,128,185, 41,  2, 16,139,  2,  0,  0, 20,
+/* 0x0240 */   0,  0,  0,  0, 41,  1, 64,249, 32,  1, 63,214,145,  0,  0, 16,
+/* 0x0250 */  48,  2,128,185, 52,  2, 16,139,  2,  0,  0, 20,  0,  0,  0,  0,
+/* 0x0260 */ 234,  3,132, 82,106,160,186,114,138,  2,  0,185,  0,  0,128,146,
+/* 0x0270 */ 225,  3, 20,170,130,  0,128,210,145,  0,  0, 16, 48,  2,128,185,
+/* 0x0280 */  41,  2, 16,139,  2,  0,  0, 20,  0,  0,  0,  0, 41,  1, 64,249,
+/* 0x0290 */  32,  1, 63,214,224,  3, 19,170, 33,  0,128, 82,226,  3, 31,170,
+/* 0x02a0 */   0,  0,  0,148,243, 83, 65,169,253,123,194,168,  0,  0,  0, 20,
+/* 0x02b0 */   1, 64, 33,139,225, 11,190,169,227,123,  1,169,  5,  0,128, 18,
+/* 0x02c0 */   4,  0,176, 82, 26,  0,  0, 20,228,123, 65,169,225, 15,194,168,
+/* 0x02d0 */   0,  0,  1,203, 66,  0,  3,203,130,  0,  0,185,229,  3,  0,170,
+/* 0x02e0 */ 224,  3,  3,170, 97,  0,  2,139,224,  3,  5,170,192,  3, 95,214,
+/* 0x02f0 */ 132,  0,  4, 43, 68,  0,  0, 52,192,  3, 95,214,  4, 68, 64,184,
+/* 0x0300 */ 132,  0,  4, 58,192,  3, 95,214, 33,  0,128, 82,240,  3, 30,170,
+/* 0x0310 */ 248,255,255,151, 33,  0,  1, 58,246,255,255,151,163,255,255, 84,
+/* 0x0320 */   0,  2, 31,214,  3, 20, 64, 56, 67, 20,  0, 56,241,255,255,151,
+/* 0x0330 */ 162,255,255, 84,245,255,255,151, 35, 12,  0,113,  1,  0,128, 82,
+/* 0x0340 */ 163,  0,  0, 84,  5, 20, 64, 56,165, 32,  3, 42,229,  3, 37, 42,
+/* 0x0350 */ 197,251,255, 52,231,255,255,151, 33,  0,  1, 58,229,255,255,151,
+/* 0x0360 */  33,  0,  1, 58, 97,  0,  0, 84,232,255,255,151, 33,  8,  0, 17,
+/* 0x0370 */ 191,  0, 52, 49, 33, 36,129, 26, 67,200,101, 56, 33,  4,  0,113,
+/* 0x0380 */  67, 20,  0, 56,162,255,255, 84,233,255,255, 23,226, 15,190,169,
+/* 0x0390 */ 254, 11,  0,249,  7, 64, 33,139,  5,  0,128, 18,  4,  0,176, 82,
+/* 0x03a0 */  19,  0,  0, 20,227, 19,193,168,  0,  0,  7,203, 66,  0,  3,203,
+/* 0x03b0 */ 130,  0,  0,185,229,  3,  0,170,224,  3,  3,170, 97,  0,  2,139,
+/* 0x03c0 */ 224,  3,  5,170,254,  7, 65,248,192,  3, 95,214,132,  0,  4, 43,
+/* 0x03d0 */  68,  0,  0, 52,192,  3, 95,214,  4, 68, 64,184,132,  0,  4, 58,
+/* 0x03e0 */ 192,  3, 95,214,  3, 20, 64, 56, 67, 20,  0, 56,248,255,255,151,
+/* 0x03f0 */ 162,255,255, 84, 33,  0,128, 82,  4,  0,  0, 20, 33,  4,  0, 81,
+/* 0x0400 */ 243,255,255,151, 33,  0,  1, 58,241,255,255,151, 33,  0,  1, 58,
+/* 0x0410 */ 239,255,255,151, 67,255,255, 84, 35, 12,  0,113,  1,  0,128, 82,
+/* 0x0420 */  35,  1,  0, 84,  5, 20, 64, 56,165, 32,  3, 42,229,  3, 37, 42,
+/* 0x0430 */ 165,251,255, 52,163,  0,  1, 83,165,124,  1, 19, 99,  0,  3, 43,
+/* 0x0440 */   2,  0,  0, 20,226,255,255,151, 33,  0,  1, 58,224,255,255,151,
+/* 0x0450 */  33,  0,  1, 58,225,  0,  0, 84, 33,  0,128, 82,220,255,255,151,
+/* 0x0460 */  33,  0,  1, 58,218,255,255,151,163,255,255, 84, 33,  8,  0, 17,
+/* 0x0470 */  33,  4,  0, 17,191,  0, 20, 49, 33, 36,129, 26, 67,  0, 64, 57,
+/* 0x0480 */  67,200,101, 56, 67, 20,  0, 56, 33,  4,  0,113,161,255,255, 84,
+/* 0x0490 */ 215,255,255, 23,254, 15, 31,248,226, 15,191,169,  7, 64, 33,139,
+/* 0x04a0 */   5,  0,128, 18,  4,  0,176, 82, 19,  0,  0, 20,227, 19,193,168,
+/* 0x04b0 */   0,  0,  7,203, 66,  0,  3,203,130,  0,  0,185,229,  3,  0,170,
+/* 0x04c0 */ 224,  3,  3,170, 97,  0,  2,139,224,  3,  5,170,254,  7, 65,248,
+/* 0x04d0 */ 192,  3, 95,214,132,  0,  4, 43, 68,  0,  0, 52,192,  3, 95,214,
+/* 0x04e0 */   4, 68, 64,184,132,  0,  4, 58,192,  3, 95,214,  3, 20, 64, 56,
+/* 0x04f0 */  67, 20,  0, 56,248,255,255,151,162,255,255, 84, 33,  0,128, 82,
+/* 0x0500 */   4,  0,  0, 20, 33,  4,  0, 81,243,255,255,151, 33,  0,  1, 26,
+/* 0x0510 */ 241,255,255,151, 33,  0,  1, 26,239,255,255,151, 67,255,255, 84,
+/* 0x0520 */  35, 12,  0,113,  1,  0,128, 82, 35,  1,  0, 84,  5, 20, 64, 56,
+/* 0x0530 */ 165, 32,  3, 42,229,  3, 37, 42,165,251,255, 52,191,  0,  0,114,
+/* 0x0540 */ 165,124,  1, 19,161,  1,  0, 84,  3,  0,  0, 20,226,255,255,151,
+/* 0x0550 */  66,  1,  0, 84, 33,  0,128, 82,223,255,255,151,226,  0,  0, 84,
+/* 0x0560 */ 221,255,255,151, 33,  0,  1, 26,219,255,255,151,163,255,255, 84,
+/* 0x0570 */  33, 16,  0, 17,  4,  0,  0, 20,215,255,255,151, 33,  0,  1, 26,
+/* 0x0580 */  33,  8,  0, 17,191,  0, 20, 49, 33, 36,129, 26, 67, 64, 33,139,
+/* 0x0590 */  99,240, 95, 56, 67,200,101, 56, 67, 20,  0, 56, 33,  4,  0,113,
+/* 0x05a0 */ 161,255,255, 84,212,255,255, 23,192,  3, 95,214,253,123,189,169,
+/* 0x05b0 */ 243, 83,  1,169,245, 91,  2,169,243,  3,  0,170,244,  3,  1,170,
+/* 0x05c0 */ 245,  3,  2,170,145,  0,  0, 16, 48,  2,128,185, 54,  2, 16,139,
+/* 0x05d0 */   2,  0,  0, 20,  0,  0,  0,  0,201,134, 64,248,201,  0,  0,180,
+/* 0x05e0 */ 224,  3, 19,170,225,  3, 20, 42,226,  3, 21,170, 32,  1, 63,214,
+/* 0x05f0 */ 250,255,255, 23,245, 91, 66,169,243, 83, 65,169,253,123,195,168,
+/* 0x0600 */ 192,  3, 95,214,  0,  0,  0,  0,  0,  0,  0,  0, 85, 80, 88, 33,
+/* 0x0610 */ 161,216,208,213,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
+/* 0x0620 */   0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 45,102,105,108,101,
+/* 0x0630 */  32,102,111,114,109, 97,116, 32,101,108,102, 54, 52, 45,108,105,
+/* 0x0640 */ 116,116,108,101, 97, 97,114, 99,104, 54, 52, 10, 10, 83,101, 99,
+/* 0x0650 */ 116,105,111,110,115, 58, 10, 73,100,120, 32, 78, 97,109,101, 32,
+/* 0x0660 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 83,105,122,101, 32, 32, 32,
+/* 0x0670 */  32, 32, 32, 86, 77, 65, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x0680 */  32, 32, 32, 32, 32, 76, 77, 65, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x0690 */  32, 32, 32, 32, 32, 32, 32, 70,105,108,101, 32,111,102,102, 32,
+/* 0x06a0 */  32, 65,108,103,110, 32, 32, 70,108, 97,103,115, 10, 32, 32, 48,
+/* 0x06b0 */  32, 83, 84, 65, 82, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 48,
+/* 0x06c0 */  32, 32, 48, 32, 32, 48, 32, 32, 48, 52, 48, 32, 32, 50, 42, 42,
+/* 0x06d0 */  48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 32, 49, 32,
+/* 0x06e0 */  80, 69, 73, 83, 68, 76, 76, 49, 32, 32, 32, 32, 32, 32, 48, 99,
+/* 0x06f0 */  32, 32, 48, 32, 32, 48, 32, 32, 48, 52, 48, 32, 32, 50, 42, 42,
+/* 0x0700 */  48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 32, 50, 32,
+/* 0x0710 */  80, 69, 77, 65, 73, 78, 48, 49, 32, 32, 32, 32, 32, 32, 48, 51,
+/* 0x0720 */  52, 32, 32, 48, 32, 32, 48, 32, 32, 48, 52, 99, 32, 32, 50, 42,
+/* 0x0730 */  42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 32, 51,
+/* 0x0740 */  32, 80, 69, 84, 76, 83, 72, 65, 75, 32, 32, 32, 32, 32, 32, 48,
+/* 0x0750 */  51, 56, 32, 32, 48, 32, 32, 48, 32, 32, 48, 56, 48, 32, 32, 50,
+/* 0x0760 */  42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 32,
+/* 0x0770 */  52, 32, 80, 69, 77, 65, 73, 78, 48, 50, 32, 32, 32, 32, 32, 32,
+/* 0x0780 */  48, 50, 56, 32, 32, 48, 32, 32, 48, 32, 32, 48, 98, 56, 32, 32,
+/* 0x0790 */  50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32,
+/* 0x07a0 */  32, 53, 32, 80, 69, 67, 65, 76, 76, 50, 66, 32, 32, 32, 32, 32,
+/* 0x07b0 */  32, 48, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48,101, 48, 32, 32,
+/* 0x07c0 */  50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32,
+/* 0x07d0 */  32, 54, 32, 80, 69, 67, 65, 76, 76, 50, 68, 32, 32, 32, 32, 32,
+/* 0x07e0 */  32, 48, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48,101, 52, 32, 32,
+/* 0x07f0 */  50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32,
+/* 0x0800 */  32, 55, 32, 80, 69, 67, 65, 76, 76, 50, 69, 32, 32, 32, 32, 32,
+/* 0x0810 */  32, 48, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48,101, 56, 32, 32,
+/* 0x0820 */  50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32,
+/* 0x0830 */  32, 56, 32, 80, 69, 77, 65, 73, 78, 49, 48, 32, 32, 32, 32, 32,
+/* 0x0840 */  32, 48, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48,101, 99, 32, 32,
+/* 0x0850 */  50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32,
+/* 0x0860 */  32, 57, 32, 80, 69, 84, 76, 83, 72, 65, 75, 50, 32, 32, 32, 32,
+/* 0x0870 */  32, 48, 56, 32, 32, 48, 32, 32, 48, 32, 32, 48,102, 48, 32, 32,
+/* 0x0880 */  50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32,
+/* 0x0890 */  49, 48, 32, 80, 69, 73, 77, 80, 79, 82, 84, 32, 32, 32, 32, 32,
+/* 0x08a0 */  32, 48, 98, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48,102, 56, 32,
+/* 0x08b0 */  32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10,
+/* 0x08c0 */  32, 49, 49, 32, 80, 69, 82, 69, 76, 79, 67, 49, 32, 32, 32, 32,
+/* 0x08d0 */  32, 32, 48, 54, 56, 32, 32, 48, 32, 32, 48, 32, 32, 48, 49, 97,
+/* 0x08e0 */  99, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84,
+/* 0x08f0 */  83, 10, 32, 49, 50, 32, 80, 69, 67, 65, 67, 72, 69, 32, 32, 32,
+/* 0x0900 */  32, 32, 32, 32, 48, 51, 56, 32, 32, 48, 32, 32, 48, 32, 32, 48,
+/* 0x0910 */  50, 49, 52, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69,
+/* 0x0920 */  78, 84, 83, 10, 32, 49, 51, 32, 80, 69, 84, 76, 83, 67, 32, 32,
+/* 0x0930 */  32, 32, 32, 32, 32, 32, 48, 53, 56, 32, 32, 48, 32, 32, 48, 32,
+/* 0x0940 */  32, 48, 50, 52, 99, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78,
+/* 0x0950 */  84, 69, 78, 84, 83, 10, 32, 49, 52, 32, 80, 69, 68, 79, 74, 85,
+/* 0x0960 */  77, 80, 32, 32, 32, 32, 32, 32, 48, 99, 32, 32, 48, 32, 32, 48,
+/* 0x0970 */  32, 32, 48, 50, 97, 52, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79,
+/* 0x0980 */  78, 84, 69, 78, 84, 83, 10, 32, 49, 53, 32, 78, 82, 86, 95, 72,
+/* 0x0990 */  69, 65, 68, 32, 32, 32, 32, 32, 32, 48, 32, 32, 48, 32, 32, 48,
+/* 0x09a0 */  32, 32, 48, 50, 98, 48, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79,
+/* 0x09b0 */  78, 84, 69, 78, 84, 83, 10, 32, 49, 54, 32, 78, 82, 86, 50, 66,
+/* 0x09c0 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 48,100, 99, 32, 32, 48, 32,
+/* 0x09d0 */  32, 48, 32, 32, 48, 50, 98, 48, 32, 32, 50, 42, 42, 48, 32, 32,
+/* 0x09e0 */  67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 49, 55, 32, 78, 82, 86,
+/* 0x09f0 */  50, 68, 32, 32, 32, 32, 32, 32, 32, 32, 32, 48, 49, 48, 56, 32,
+/* 0x0a00 */  32, 48, 32, 32, 48, 32, 32, 48, 51, 56, 99, 32, 32, 50, 42, 42,
+/* 0x0a10 */  48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32, 49, 56, 32,
+/* 0x0a20 */  78, 82, 86, 50, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 48, 49,
+/* 0x0a30 */  49, 52, 32, 32, 48, 32, 32, 48, 32, 32, 48, 52, 57, 52, 32, 32,
+/* 0x0a40 */  50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83, 10, 32,
+/* 0x0a50 */  49, 57, 32, 80, 69, 84, 76, 83, 67, 50, 32, 32, 32, 32, 32, 32,
+/* 0x0a60 */  32, 48, 53, 99, 32, 32, 48, 32, 32, 48, 32, 32, 48, 53, 97, 56,
+/* 0x0a70 */  32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84, 83,
+/* 0x0a80 */  10, 32, 50, 48, 32, 80, 69, 70, 73, 76, 84, 83, 89, 77, 32, 32,
+/* 0x0a90 */  32, 32, 32, 48, 56, 32, 32, 48, 32, 32, 48, 32, 32, 48, 54, 48,
+/* 0x0aa0 */  52, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69, 78, 84,
+/* 0x0ab0 */  83, 10, 32, 50, 49, 32, 85, 80, 88, 49, 72, 69, 65, 68, 32, 32,
+/* 0x0ac0 */  32, 32, 32, 32, 48, 50, 48, 32, 32, 48, 32, 32, 48, 32, 32, 48,
+/* 0x0ad0 */  54, 48, 99, 32, 32, 50, 42, 42, 48, 32, 32, 67, 79, 78, 84, 69,
+/* 0x0ae0 */  78, 84, 83, 10, 83, 89, 77, 66, 79, 76, 32, 84, 65, 66, 76, 69,
+/* 0x0af0 */  58, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0b00 */  48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 84, 76, 83,
+/* 0x0b10 */  67, 50, 32, 48, 32, 80, 69, 84, 76, 83, 67, 50, 10, 48, 48, 48,
+/* 0x0b20 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32,
+/* 0x0b30 */  32, 32, 32,100, 32, 32, 83, 84, 65, 82, 84, 32, 48, 32, 83, 84,
+/* 0x0b40 */  65, 82, 84, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0b50 */  48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 73,
+/* 0x0b60 */  83, 68, 76, 76, 49, 32, 48, 32, 80, 69, 73, 83, 68, 76, 76, 49,
+/* 0x0b70 */  10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0b80 */  48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 77, 65, 73, 78,
+/* 0x0b90 */  48, 49, 32, 48, 32, 80, 69, 77, 65, 73, 78, 48, 49, 10, 48, 48,
+/* 0x0ba0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108,
+/* 0x0bb0 */  32, 32, 32, 32,100, 32, 32, 80, 69, 84, 76, 83, 72, 65, 75, 32,
+/* 0x0bc0 */  48, 32, 80, 69, 84, 76, 83, 72, 65, 75, 10, 48, 48, 48, 48, 48,
+/* 0x0bd0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32,
+/* 0x0be0 */  32,100, 32, 32, 80, 69, 77, 65, 73, 78, 48, 50, 32, 48, 32, 80,
+/* 0x0bf0 */  69, 77, 65, 73, 78, 48, 50, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0c00 */  48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32,
+/* 0x0c10 */  32, 80, 69, 67, 65, 76, 76, 50, 66, 32, 48, 32, 80, 69, 67, 65,
+/* 0x0c20 */  76, 76, 50, 66, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0c30 */  48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69,
+/* 0x0c40 */  67, 65, 76, 76, 50, 68, 32, 48, 32, 80, 69, 67, 65, 76, 76, 50,
+/* 0x0c50 */  68, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0c60 */  48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 67, 65, 76,
+/* 0x0c70 */  76, 50, 69, 32, 48, 32, 80, 69, 67, 65, 76, 76, 50, 69, 10, 48,
+/* 0x0c80 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,
+/* 0x0c90 */ 108, 32, 32, 32, 32,100, 32, 32, 80, 69, 77, 65, 73, 78, 49, 48,
+/* 0x0ca0 */  32, 48, 32, 80, 69, 77, 65, 73, 78, 49, 48, 10, 48, 48, 48, 48,
+/* 0x0cb0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32,
+/* 0x0cc0 */  32, 32,100, 32, 32, 80, 69, 84, 76, 83, 72, 65, 75, 50, 32, 48,
+/* 0x0cd0 */  32, 80, 69, 84, 76, 83, 72, 65, 75, 50, 10, 48, 48, 48, 48, 48,
+/* 0x0ce0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32,
+/* 0x0cf0 */  32,100, 32, 32, 80, 69, 73, 77, 80, 79, 82, 84, 32, 48, 32, 80,
+/* 0x0d00 */  69, 73, 77, 80, 79, 82, 84, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0d10 */  48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32,
+/* 0x0d20 */  32, 80, 69, 82, 69, 76, 79, 67, 49, 32, 48, 32, 80, 69, 82, 69,
+/* 0x0d30 */  76, 79, 67, 49, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0d40 */  48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 80, 69,
+/* 0x0d50 */  67, 65, 67, 72, 69, 32, 48, 32, 80, 69, 67, 65, 67, 72, 69, 10,
+/* 0x0d60 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0d70 */  32,108, 32, 32, 32, 32,100, 32, 32, 80, 69, 84, 76, 83, 67, 32,
+/* 0x0d80 */  48, 32, 80, 69, 84, 76, 83, 67, 10, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0d90 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100,
+/* 0x0da0 */  32, 32, 80, 69, 68, 79, 74, 85, 77, 80, 32, 48, 32, 80, 69, 68,
+/* 0x0db0 */  79, 74, 85, 77, 80, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0dc0 */  48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 78,
+/* 0x0dd0 */  82, 86, 95, 72, 69, 65, 68, 32, 48, 32, 78, 82, 86, 95, 72, 69,
+/* 0x0de0 */  65, 68, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0df0 */  48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32, 78, 82, 86, 50,
+/* 0x0e00 */  66, 32, 48, 32, 78, 82, 86, 50, 66, 10, 48, 48, 48, 48, 48, 48,
+/* 0x0e10 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,
+/* 0x0e20 */ 100, 32, 32, 78, 82, 86, 50, 68, 32, 48, 32, 78, 82, 86, 50, 68,
+/* 0x0e30 */  10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0e40 */  48, 32,108, 32, 32, 32, 32,100, 32, 32, 78, 82, 86, 50, 69, 32,
+/* 0x0e50 */  48, 32, 78, 82, 86, 50, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0e60 */  48, 48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32,
+/* 0x0e70 */  32, 80, 69, 70, 73, 76, 84, 83, 89, 77, 32, 48, 32, 80, 69, 70,
+/* 0x0e80 */  73, 76, 84, 83, 89, 77, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0e90 */  48, 48, 48, 48, 48, 48, 48, 32,108, 32, 32, 32, 32,100, 32, 32,
+/* 0x0ea0 */  85, 80, 88, 49, 72, 69, 65, 68, 32, 48, 32, 85, 80, 88, 49, 72,
+/* 0x0eb0 */  69, 65, 68, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0ec0 */  48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78,
+/* 0x0ed0 */  68, 42, 32, 48, 32,111,114,105,103,105,110, 97,108, 95,101,110,
+/* 0x0ee0 */ 116,114,121, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0ef0 */  48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78,
+/* 0x0f00 */  68, 42, 32, 48, 32,115,116, 97,114,116, 95,111,102, 95, 99,111,
+/* 0x0f10 */ 109,112,114,101,115,115,101,100, 10, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0f20 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32,
+/* 0x0f30 */  32, 32, 42, 85, 78, 68, 42, 32, 48, 32,115,116, 97,114,116, 95,
+/* 0x0f40 */ 111,102, 95,117,110, 99,111,109,112,114,101,115,115,101,100, 10,
+/* 0x0f50 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0f60 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48,
+/* 0x0f70 */  32,116,108,115, 95, 97,100,100,114,101,115,115, 10, 48, 48, 48,
+/* 0x0f80 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32,
+/* 0x0f90 */  32, 32, 32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32,116,108,
+/* 0x0fa0 */ 115, 95,118, 97,108,117,101, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0fb0 */  48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x0fc0 */  32, 42, 85, 78, 68, 42, 32, 48, 32, 99,111,109,112, 95,108,101,
+/* 0x0fd0 */ 110, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x0fe0 */  48, 48, 32,103, 32, 32, 32, 32, 32, 70, 32, 78, 82, 86, 50, 66,
+/* 0x0ff0 */  32, 48,100, 99, 32,117, 99,108, 95,110,114,118, 50, 98, 95,100,
+/* 0x1000 */ 101, 99,111,109,112,114,101,115,115, 95, 51, 50, 10, 48, 48, 48,
+/* 0x1010 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,103, 32,
+/* 0x1020 */  32, 32, 32, 32, 70, 32, 78, 82, 86, 50, 68, 32, 48, 49, 48, 56,
+/* 0x1030 */  32,117, 99,108, 95,110,114,118, 50,100, 95,100,101, 99,111,109,
+/* 0x1040 */ 112,114,101,115,115, 95, 51, 50, 10, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1050 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 32,103, 32, 32, 32, 32, 32,
+/* 0x1060 */  70, 32, 78, 82, 86, 50, 69, 32, 48, 49, 49, 52, 32,117, 99,108,
+/* 0x1070 */  95,110,114,118, 50,101, 95,100,101, 99,111,109,112,114,101,115,
+/* 0x1080 */ 115, 95, 51, 50, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1090 */  48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85,
+/* 0x10a0 */  78, 68, 42, 32, 48, 32, 99,111,109,112,114,101,115,115,101,100,
+/* 0x10b0 */  95,105,109,112,111,114,116,115, 10, 48, 48, 48, 48, 48, 48, 48,
+/* 0x10c0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32,
+/* 0x10d0 */  32, 32, 42, 85, 78, 68, 42, 32, 48, 32,115,116, 97,114,116, 95,
+/* 0x10e0 */ 111,102, 95,105,109,112,111,114,116,115, 10, 48, 48, 48, 48, 48,
+/* 0x10f0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32,
+/* 0x1100 */  32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32, 76,111, 97,100,
+/* 0x1110 */  76,105, 98,114, 97,114,121, 65, 10, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1120 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1130 */  32, 32, 42, 85, 78, 68, 42, 32, 48, 32, 71,101,116, 80,114,111,
+/* 0x1140 */  99, 65,100,100,114,101,115,115, 10, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1150 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1160 */  32, 32, 42, 85, 78, 68, 42, 32, 48, 32,115,116, 97,114,116, 95,
+/* 0x1170 */ 111,102, 95,114,101,108,111, 99,115, 10, 48, 48, 48, 48, 48, 48,
+/* 0x1180 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32,
+/* 0x1190 */  32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32,115,105,122,101,111,
+/* 0x11a0 */ 102, 95,105,109, 97,103,101, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x11b0 */  48, 48, 48, 48, 48, 48, 48, 48, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x11c0 */  32, 42, 85, 78, 68, 42, 32, 48, 32, 70,108,117,115,104, 73,110,
+/* 0x11d0 */ 115,116,114,117, 99,116,105,111,110, 67, 97, 99,104,101, 10, 48,
+/* 0x11e0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32,
+/* 0x11f0 */  32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78, 68, 42, 32, 48, 32,
+/* 0x1200 */ 116,108,115, 95, 99, 97,108,108, 98, 97, 99,107,115, 95,112,116,
+/* 0x1210 */ 114, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1220 */  48, 48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78, 68, 42,
+/* 0x1230 */  32, 48, 32,102,105,108,116,101,114, 95,108,101,110,103,116,104,
+/* 0x1240 */  10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1250 */  48, 32, 32, 32, 32, 32, 32, 32, 32, 32, 42, 85, 78, 68, 42, 32,
+/* 0x1260 */  48, 32,102,105,108,116,101,114, 95, 99,116,111, 10, 10, 82, 69,
 /* 0x1270 */  76, 79, 67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83,
-/* 0x1280 */  32, 70, 79, 82, 32, 91, 80, 69, 67, 65, 76, 76, 50, 68, 93, 58,
+/* 0x1280 */  32, 70, 79, 82, 32, 91, 80, 69, 73, 83, 68, 76, 76, 49, 93, 58,
 /* 0x1290 */  10, 79, 70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32,
 /* 0x12a0 */  32, 32, 84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
 /* 0x12b0 */  32, 32, 32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48,
-/* 0x12c0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 82, 95, 65, 65, 82,
-/* 0x12d0 */  67, 72, 54, 52, 95, 67, 65, 76, 76, 50, 54, 32, 32,117, 99,108,
-/* 0x12e0 */  95,110,114,118, 50,100, 95,100,101, 99,111,109,112,114,101,115,
-/* 0x12f0 */ 115, 95, 51, 50, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79, 78,
-/* 0x1300 */  32, 82, 69, 67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80, 69,
-/* 0x1310 */  67, 65, 76, 76, 50, 69, 93, 58, 10, 79, 70, 70, 83, 69, 84, 32,
-/* 0x1320 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32, 32,
-/* 0x1330 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76, 85,
-/* 0x1340 */  69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1350 */  48, 48, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 67, 65, 76,
-/* 0x1360 */  76, 50, 54, 32, 32,117, 99,108, 95,110,114,118, 50,101, 95,100,
-/* 0x1370 */ 101, 99,111,109,112,114,101,115,115, 95, 51, 50, 10, 10, 82, 69,
-/* 0x1380 */  76, 79, 67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83,
-/* 0x1390 */  32, 70, 79, 82, 32, 91, 80, 69, 73, 77, 80, 79, 82, 84, 93, 58,
-/* 0x13a0 */  10, 79, 70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-/* 0x13b0 */  32, 32, 84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-/* 0x13c0 */  32, 32, 32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48,
-/* 0x13d0 */  48, 48, 48, 48, 48, 48, 48, 48, 49, 48, 32, 82, 95, 65, 65, 82,
-/* 0x13e0 */  67, 72, 54, 52, 95, 65, 66, 83, 51, 50, 32, 32, 32, 99,111,109,
-/* 0x13f0 */ 112,114,101,115,115,101,100, 95,105,109,112,111,114,116,115, 10,
-/* 0x1400 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 51, 52,
-/* 0x1410 */  32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 65, 66, 83, 51, 50,
-/* 0x1420 */  32, 32, 32,115,116, 97,114,116, 95,111,102, 95,105,109,112,111,
-/* 0x1430 */ 114,116,115, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1440 */  48, 48, 53, 56, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 80,
-/* 0x1450 */  82, 69, 76, 51, 50, 32, 32, 76,111, 97,100, 76,105, 98,114, 97,
-/* 0x1460 */ 114,121, 65, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1470 */  48, 48, 97, 48, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 80,
-/* 0x1480 */  82, 69, 76, 51, 50, 32, 32, 71,101,116, 80,114,111, 99, 65,100,
-/* 0x1490 */ 100,114,101,115,115, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79,
-/* 0x14a0 */  78, 32, 82, 69, 67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80,
-/* 0x14b0 */  69, 82, 69, 76, 79, 67, 49, 93, 58, 10, 79, 70, 70, 83, 69, 84,
-/* 0x14c0 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32,
-/* 0x14d0 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76,
-/* 0x14e0 */  85, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x14f0 */  48, 49, 48, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 65, 66,
-/* 0x1500 */  83, 51, 50, 32, 32, 32,115,116, 97,114,116, 95,111,102, 95,114,
-/* 0x1510 */ 101,108,111, 99,115, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79,
-/* 0x1520 */  78, 32, 82, 69, 67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80,
-/* 0x1530 */  69, 67, 65, 67, 72, 69, 93, 58, 10, 79, 70, 70, 83, 69, 84, 32,
-/* 0x1540 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32, 32,
-/* 0x1550 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76, 85,
-/* 0x1560 */  69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1570 */  49, 56, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 65, 66, 83,
-/* 0x1580 */  51, 50, 32, 32, 32,115,105,122,101,111,102, 95,105,109, 97,103,
-/* 0x1590 */ 101, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x15a0 */  50, 99, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 80, 82, 69,
-/* 0x15b0 */  76, 51, 50, 32, 32, 70,108,117,115,104, 73,110,115,116,114,117,
-/* 0x15c0 */  99,116,105,111,110, 67, 97, 99,104,101, 10, 10, 82, 69, 76, 79,
-/* 0x15d0 */  67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32, 70,
-/* 0x15e0 */  79, 82, 32, 91, 80, 69, 84, 76, 83, 67, 93, 58, 10, 79, 70, 70,
-/* 0x15f0 */  83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 84, 89,
-/* 0x1600 */  80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
-/* 0x1610 */  86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1620 */  48, 48, 48, 48, 49, 48, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52,
-/* 0x1630 */  95, 80, 82, 69, 76, 51, 50, 32, 32, 80, 69, 84, 76, 83, 67, 50,
-/* 0x1640 */  10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 51,
-/* 0x1650 */  99, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 80, 82, 69, 76,
-/* 0x1660 */  51, 50, 32, 32, 70,108,117,115,104, 73,110,115,116,114,117, 99,
-/* 0x1670 */ 116,105,111,110, 67, 97, 99,104,101, 10, 48, 48, 48, 48, 48, 48,
-/* 0x1680 */  48, 48, 48, 48, 48, 48, 48, 48, 53, 52, 32, 82, 95, 65, 65, 82,
-/* 0x1690 */  67, 72, 54, 52, 95, 67, 65, 76, 76, 50, 54, 32, 32, 80, 69, 84,
-/* 0x16a0 */  76, 83, 67, 50, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79, 78,
-/* 0x16b0 */  32, 82, 69, 67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80, 69,
-/* 0x16c0 */  68, 79, 74, 85, 77, 80, 93, 58, 10, 79, 70, 70, 83, 69, 84, 32,
-/* 0x16d0 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32, 32,
-/* 0x16e0 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76, 85,
-/* 0x16f0 */  69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1700 */  48, 56, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 74, 85, 77,
-/* 0x1710 */  80, 50, 54, 32, 32,111,114,105,103,105,110, 97,108, 95,101,110,
-/* 0x1720 */ 116,114,121, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79, 78, 32,
-/* 0x1730 */  82, 69, 67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80, 69, 84,
-/* 0x1740 */  76, 83, 67, 50, 93, 58, 10, 79, 70, 70, 83, 69, 84, 32, 32, 32,
-/* 0x1750 */  32, 32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32, 32, 32, 32,
-/* 0x1760 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76, 85, 69, 10,
-/* 0x1770 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 50, 99,
-/* 0x1780 */  32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 80, 82, 69, 76, 51,
-/* 0x1790 */  50, 32, 32,116,108,115, 95, 99, 97,108,108, 98, 97, 99,107,115,
-/* 0x17a0 */  95,112,116,114, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79, 78,
-/* 0x17b0 */  32, 82, 69, 67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80, 69,
-/* 0x17c0 */  70, 73, 76, 84, 83, 89, 77, 93, 58, 10, 79, 70, 70, 83, 69, 84,
-/* 0x17d0 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32,
-/* 0x17e0 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76,
-/* 0x17f0 */  85, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1800 */  48, 48, 48, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 65, 66,
-/* 0x1810 */  83, 51, 50, 32, 32, 32,102,105,108,116,101,114, 95,108,101,110,
-/* 0x1820 */ 103,116,104, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
-/* 0x1830 */  48, 48, 48, 52, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 65,
-/* 0x1840 */  66, 83, 51, 50, 32, 32, 32,102,105,108,116,101,114, 95, 99,116,
-/* 0x1850 */ 111, 10
+/* 0x12c0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 56, 32, 82, 95, 65, 65, 82,
+/* 0x12d0 */  67, 72, 54, 52, 95, 74, 85, 77, 80, 50, 54, 32, 32,111,114,105,
+/* 0x12e0 */ 103,105,110, 97,108, 95,101,110,116,114,121, 10, 10, 82, 69, 76,
+/* 0x12f0 */  79, 67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32,
+/* 0x1300 */  70, 79, 82, 32, 91, 80, 69, 77, 65, 73, 78, 48, 49, 93, 58, 10,
+/* 0x1310 */  79, 70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1320 */  32, 84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1330 */  32, 32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1340 */  48, 48, 48, 48, 48, 48, 48, 49, 56, 32, 82, 95, 65, 65, 82, 67,
+/* 0x1350 */  72, 54, 52, 95, 80, 82, 69, 76, 51, 50, 32, 32,115,116, 97,114,
+/* 0x1360 */ 116, 95,111,102, 95, 99,111,109,112,114,101,115,115,101,100, 10,
+/* 0x1370 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 50, 99,
+/* 0x1380 */  32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 65, 66, 83, 51, 50,
+/* 0x1390 */  32, 32, 32,115,116, 97,114,116, 95,111,102, 95,117,110, 99,111,
+/* 0x13a0 */ 109,112,114,101,115,115,101,100, 10, 10, 82, 69, 76, 79, 67, 65,
+/* 0x13b0 */  84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32, 70, 79, 82,
+/* 0x13c0 */  32, 91, 80, 69, 84, 76, 83, 72, 65, 75, 93, 58, 10, 79, 70, 70,
+/* 0x13d0 */  83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 84, 89,
+/* 0x13e0 */  80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x13f0 */  86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1400 */  48, 48, 48, 48, 49, 48, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52,
+/* 0x1410 */  95, 65, 66, 83, 51, 50, 32, 32, 32,116,108,115, 95, 97,100,100,
+/* 0x1420 */ 114,101,115,115, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1430 */  48, 48, 48, 51, 48, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95,
+/* 0x1440 */  65, 66, 83, 51, 50, 32, 32, 32,116,108,115, 95,118, 97,108,117,
+/* 0x1450 */ 101, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79, 78, 32, 82, 69,
+/* 0x1460 */  67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80, 69, 77, 65, 73,
+/* 0x1470 */  78, 48, 50, 93, 58, 10, 79, 70, 70, 83, 69, 84, 32, 32, 32, 32,
+/* 0x1480 */  32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32, 32, 32, 32, 32,
+/* 0x1490 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76, 85, 69, 10, 48,
+/* 0x14a0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 49, 52, 32,
+/* 0x14b0 */  82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 65, 66, 83, 51, 50, 32,
+/* 0x14c0 */  32, 32, 99,111,109,112, 95,108,101,110, 10, 10, 82, 69, 76, 79,
+/* 0x14d0 */  67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32, 70,
+/* 0x14e0 */  79, 82, 32, 91, 80, 69, 67, 65, 76, 76, 50, 66, 93, 58, 10, 79,
+/* 0x14f0 */  70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1500 */  84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1510 */  32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1520 */  48, 48, 48, 48, 48, 48, 48, 48, 32, 82, 95, 65, 65, 82, 67, 72,
+/* 0x1530 */  54, 52, 95, 67, 65, 76, 76, 50, 54, 32, 32,117, 99,108, 95,110,
+/* 0x1540 */ 114,118, 50, 98, 95,100,101, 99,111,109,112,114,101,115,115, 95,
+/* 0x1550 */  51, 50, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79, 78, 32, 82,
+/* 0x1560 */  69, 67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80, 69, 67, 65,
+/* 0x1570 */  76, 76, 50, 68, 93, 58, 10, 79, 70, 70, 83, 69, 84, 32, 32, 32,
+/* 0x1580 */  32, 32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32, 32, 32, 32,
+/* 0x1590 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76, 85, 69, 10,
+/* 0x15a0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x15b0 */  32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 67, 65, 76, 76, 50,
+/* 0x15c0 */  54, 32, 32,117, 99,108, 95,110,114,118, 50,100, 95,100,101, 99,
+/* 0x15d0 */ 111,109,112,114,101,115,115, 95, 51, 50, 10, 10, 82, 69, 76, 79,
+/* 0x15e0 */  67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32, 70,
+/* 0x15f0 */  79, 82, 32, 91, 80, 69, 67, 65, 76, 76, 50, 69, 93, 58, 10, 79,
+/* 0x1600 */  70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1610 */  84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1620 */  32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1630 */  48, 48, 48, 48, 48, 48, 48, 48, 32, 82, 95, 65, 65, 82, 67, 72,
+/* 0x1640 */  54, 52, 95, 67, 65, 76, 76, 50, 54, 32, 32,117, 99,108, 95,110,
+/* 0x1650 */ 114,118, 50,101, 95,100,101, 99,111,109,112,114,101,115,115, 95,
+/* 0x1660 */  51, 50, 10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79, 78, 32, 82,
+/* 0x1670 */  69, 67, 79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80, 69, 73, 77,
+/* 0x1680 */  80, 79, 82, 84, 93, 58, 10, 79, 70, 70, 83, 69, 84, 32, 32, 32,
+/* 0x1690 */  32, 32, 32, 32, 32, 32, 32, 32, 84, 89, 80, 69, 32, 32, 32, 32,
+/* 0x16a0 */  32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 86, 65, 76, 85, 69, 10,
+/* 0x16b0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 49, 48,
+/* 0x16c0 */  32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 65, 66, 83, 51, 50,
+/* 0x16d0 */  32, 32, 32, 99,111,109,112,114,101,115,115,101,100, 95,105,109,
+/* 0x16e0 */ 112,111,114,116,115, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x16f0 */  48, 48, 48, 48, 51, 52, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52,
+/* 0x1700 */  95, 65, 66, 83, 51, 50, 32, 32, 32,115,116, 97,114,116, 95,111,
+/* 0x1710 */ 102, 95,105,109,112,111,114,116,115, 10, 48, 48, 48, 48, 48, 48,
+/* 0x1720 */  48, 48, 48, 48, 48, 48, 48, 48, 53, 56, 32, 82, 95, 65, 65, 82,
+/* 0x1730 */  67, 72, 54, 52, 95, 80, 82, 69, 76, 51, 50, 32, 32, 76,111, 97,
+/* 0x1740 */ 100, 76,105, 98,114, 97,114,121, 65, 10, 48, 48, 48, 48, 48, 48,
+/* 0x1750 */  48, 48, 48, 48, 48, 48, 48, 48, 97, 48, 32, 82, 95, 65, 65, 82,
+/* 0x1760 */  67, 72, 54, 52, 95, 80, 82, 69, 76, 51, 50, 32, 32, 71,101,116,
+/* 0x1770 */  80,114,111, 99, 65,100,100,114,101,115,115, 10, 10, 82, 69, 76,
+/* 0x1780 */  79, 67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32,
+/* 0x1790 */  70, 79, 82, 32, 91, 80, 69, 82, 69, 76, 79, 67, 49, 93, 58, 10,
+/* 0x17a0 */  79, 70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x17b0 */  32, 84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x17c0 */  32, 32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48,
+/* 0x17d0 */  48, 48, 48, 48, 48, 48, 48, 49, 48, 32, 82, 95, 65, 65, 82, 67,
+/* 0x17e0 */  72, 54, 52, 95, 65, 66, 83, 51, 50, 32, 32, 32,115,116, 97,114,
+/* 0x17f0 */ 116, 95,111,102, 95,114,101,108,111, 99,115, 10, 10, 82, 69, 76,
+/* 0x1800 */  79, 67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32,
+/* 0x1810 */  70, 79, 82, 32, 91, 80, 69, 67, 65, 67, 72, 69, 93, 58, 10, 79,
+/* 0x1820 */  70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1830 */  84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1840 */  32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1850 */  48, 48, 48, 48, 48, 48, 49, 56, 32, 82, 95, 65, 65, 82, 67, 72,
+/* 0x1860 */  54, 52, 95, 65, 66, 83, 51, 50, 32, 32, 32,115,105,122,101,111,
+/* 0x1870 */ 102, 95,105,109, 97,103,101, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1880 */  48, 48, 48, 48, 48, 48, 50, 99, 32, 82, 95, 65, 65, 82, 67, 72,
+/* 0x1890 */  54, 52, 95, 80, 82, 69, 76, 51, 50, 32, 32, 70,108,117,115,104,
+/* 0x18a0 */  73,110,115,116,114,117, 99,116,105,111,110, 67, 97, 99,104,101,
+/* 0x18b0 */  10, 10, 82, 69, 76, 79, 67, 65, 84, 73, 79, 78, 32, 82, 69, 67,
+/* 0x18c0 */  79, 82, 68, 83, 32, 70, 79, 82, 32, 91, 80, 69, 84, 76, 83, 67,
+/* 0x18d0 */  93, 58, 10, 79, 70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32,
+/* 0x18e0 */  32, 32, 32, 32, 84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x18f0 */  32, 32, 32, 32, 32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48,
+/* 0x1900 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 49, 48, 32, 82, 95, 65,
+/* 0x1910 */  65, 82, 67, 72, 54, 52, 95, 80, 82, 69, 76, 51, 50, 32, 32, 80,
+/* 0x1920 */  69, 84, 76, 83, 67, 50, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1930 */  48, 48, 48, 48, 48, 51, 99, 32, 82, 95, 65, 65, 82, 67, 72, 54,
+/* 0x1940 */  52, 95, 80, 82, 69, 76, 51, 50, 32, 32, 70,108,117,115,104, 73,
+/* 0x1950 */ 110,115,116,114,117, 99,116,105,111,110, 67, 97, 99,104,101, 10,
+/* 0x1960 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48, 53, 52,
+/* 0x1970 */  32, 82, 95, 65, 65, 82, 67, 72, 54, 52, 95, 67, 65, 76, 76, 50,
+/* 0x1980 */  54, 32, 32, 80, 69, 84, 76, 83, 67, 50, 10, 10, 82, 69, 76, 79,
+/* 0x1990 */  67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32, 70,
+/* 0x19a0 */  79, 82, 32, 91, 80, 69, 68, 79, 74, 85, 77, 80, 93, 58, 10, 79,
+/* 0x19b0 */  70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x19c0 */  84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x19d0 */  32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x19e0 */  48, 48, 48, 48, 48, 48, 48, 56, 32, 82, 95, 65, 65, 82, 67, 72,
+/* 0x19f0 */  54, 52, 95, 74, 85, 77, 80, 50, 54, 32, 32,111,114,105,103,105,
+/* 0x1a00 */ 110, 97,108, 95,101,110,116,114,121, 10, 10, 82, 69, 76, 79, 67,
+/* 0x1a10 */  65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32, 70, 79,
+/* 0x1a20 */  82, 32, 91, 80, 69, 84, 76, 83, 67, 50, 93, 58, 10, 79, 70, 70,
+/* 0x1a30 */  83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 84, 89,
+/* 0x1a40 */  80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1a50 */  86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1a60 */  48, 48, 48, 48, 50, 99, 32, 82, 95, 65, 65, 82, 67, 72, 54, 52,
+/* 0x1a70 */  95, 80, 82, 69, 76, 51, 50, 32, 32,116,108,115, 95, 99, 97,108,
+/* 0x1a80 */ 108, 98, 97, 99,107,115, 95,112,116,114, 10, 10, 82, 69, 76, 79,
+/* 0x1a90 */  67, 65, 84, 73, 79, 78, 32, 82, 69, 67, 79, 82, 68, 83, 32, 70,
+/* 0x1aa0 */  79, 82, 32, 91, 80, 69, 70, 73, 76, 84, 83, 89, 77, 93, 58, 10,
+/* 0x1ab0 */  79, 70, 70, 83, 69, 84, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1ac0 */  32, 84, 89, 80, 69, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32,
+/* 0x1ad0 */  32, 32, 32, 86, 65, 76, 85, 69, 10, 48, 48, 48, 48, 48, 48, 48,
+/* 0x1ae0 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 32, 82, 95, 65, 65, 82, 67,
+/* 0x1af0 */  72, 54, 52, 95, 65, 66, 83, 51, 50, 32, 32, 32,102,105,108,116,
+/* 0x1b00 */ 101,114, 95,108,101,110,103,116,104, 10, 48, 48, 48, 48, 48, 48,
+/* 0x1b10 */  48, 48, 48, 48, 48, 48, 48, 48, 48, 52, 32, 82, 95, 65, 65, 82,
+/* 0x1b20 */  67, 72, 54, 52, 95, 65, 66, 83, 51, 50, 32, 32, 32,102,105,108,
+/* 0x1b30 */ 116,101,114, 95, 99,116,111, 10
 };
diff --git a/src/stub/tmp/arm64-win64.pe.bin.dump b/src/stub/tmp/arm64-win64.pe.bin.dump
index df0c3024..50589828 100644
--- a/src/stub/tmp/arm64-win64.pe.bin.dump
+++ b/src/stub/tmp/arm64-win64.pe.bin.dump
@@ -3,33 +3,39 @@ file format elf64-littleaarch64
 Sections:
 Idx Name          Size      VMA               LMA               File off  Algn  Flags
   0 START         0  0  0  040  2**0  CONTENTS
-  1 PEMAIN01      034  0  0  040  2**0  CONTENTS
-  2 PEMAIN02      028  0  0  074  2**0  CONTENTS
-  3 PECALL2B      04  0  0  09c  2**0  CONTENTS
-  4 PECALL2D      04  0  0  0a0  2**0  CONTENTS
-  5 PECALL2E      04  0  0  0a4  2**0  CONTENTS
-  6 PEMAIN10      04  0  0  0a8  2**0  CONTENTS
-  7 PEIMPORT      0b4  0  0  0ac  2**0  CONTENTS
-  8 PERELOC1      068  0  0  0160  2**0  CONTENTS
-  9 PECACHE       038  0  0  01c8  2**0  CONTENTS
- 10 PETLSC        058  0  0  0200  2**0  CONTENTS
- 11 PEDOJUMP      0c  0  0  0258  2**0  CONTENTS
- 12 NRV_HEAD      0  0  0  0264  2**0  CONTENTS
- 13 NRV2B         0dc  0  0  0264  2**0  CONTENTS
- 14 NRV2D         0108  0  0  0340  2**0  CONTENTS
- 15 NRV2E         0114  0  0  0448  2**0  CONTENTS
- 16 PETLSC2       05c  0  0  055c  2**0  CONTENTS
- 17 PEFILTSYM     08  0  0  05b8  2**0  CONTENTS
- 18 UPX1HEAD      020  0  0  05c0  2**0  CONTENTS
+  1 PEISDLL1      0c  0  0  040  2**0  CONTENTS
+  2 PEMAIN01      034  0  0  04c  2**0  CONTENTS
+  3 PETLSHAK      038  0  0  080  2**0  CONTENTS
+  4 PEMAIN02      028  0  0  0b8  2**0  CONTENTS
+  5 PECALL2B      04  0  0  0e0  2**0  CONTENTS
+  6 PECALL2D      04  0  0  0e4  2**0  CONTENTS
+  7 PECALL2E      04  0  0  0e8  2**0  CONTENTS
+  8 PEMAIN10      04  0  0  0ec  2**0  CONTENTS
+  9 PETLSHAK2     08  0  0  0f0  2**0  CONTENTS
+ 10 PEIMPORT      0b4  0  0  0f8  2**0  CONTENTS
+ 11 PERELOC1      068  0  0  01ac  2**0  CONTENTS
+ 12 PECACHE       038  0  0  0214  2**0  CONTENTS
+ 13 PETLSC        058  0  0  024c  2**0  CONTENTS
+ 14 PEDOJUMP      0c  0  0  02a4  2**0  CONTENTS
+ 15 NRV_HEAD      0  0  0  02b0  2**0  CONTENTS
+ 16 NRV2B         0dc  0  0  02b0  2**0  CONTENTS
+ 17 NRV2D         0108  0  0  038c  2**0  CONTENTS
+ 18 NRV2E         0114  0  0  0494  2**0  CONTENTS
+ 19 PETLSC2       05c  0  0  05a8  2**0  CONTENTS
+ 20 PEFILTSYM     08  0  0  0604  2**0  CONTENTS
+ 21 UPX1HEAD      020  0  0  060c  2**0  CONTENTS
 SYMBOL TABLE:
 0000000000000000 l    d  PETLSC2 0 PETLSC2
 0000000000000000 l    d  START 0 START
+0000000000000000 l    d  PEISDLL1 0 PEISDLL1
 0000000000000000 l    d  PEMAIN01 0 PEMAIN01
+0000000000000000 l    d  PETLSHAK 0 PETLSHAK
 0000000000000000 l    d  PEMAIN02 0 PEMAIN02
 0000000000000000 l    d  PECALL2B 0 PECALL2B
 0000000000000000 l    d  PECALL2D 0 PECALL2D
 0000000000000000 l    d  PECALL2E 0 PECALL2E
 0000000000000000 l    d  PEMAIN10 0 PEMAIN10
+0000000000000000 l    d  PETLSHAK2 0 PETLSHAK2
 0000000000000000 l    d  PEIMPORT 0 PEIMPORT
 0000000000000000 l    d  PERELOC1 0 PERELOC1
 0000000000000000 l    d  PECACHE 0 PECACHE
@@ -41,8 +47,11 @@ SYMBOL TABLE:
 0000000000000000 l    d  NRV2E 0 NRV2E
 0000000000000000 l    d  PEFILTSYM 0 PEFILTSYM
 0000000000000000 l    d  UPX1HEAD 0 UPX1HEAD
+0000000000000000         *UND* 0 original_entry
 0000000000000000         *UND* 0 start_of_compressed
 0000000000000000         *UND* 0 start_of_uncompressed
+0000000000000000         *UND* 0 tls_address
+0000000000000000         *UND* 0 tls_value
 0000000000000000         *UND* 0 comp_len
 0000000000000000 g     F NRV2B 0dc ucl_nrv2b_decompress_32
 0000000000000000 g     F NRV2D 0108 ucl_nrv2d_decompress_32
@@ -54,16 +63,24 @@ SYMBOL TABLE:
 0000000000000000         *UND* 0 start_of_relocs
 0000000000000000         *UND* 0 sizeof_image
 0000000000000000         *UND* 0 FlushInstructionCache
-0000000000000000         *UND* 0 original_entry
 0000000000000000         *UND* 0 tls_callbacks_ptr
 0000000000000000         *UND* 0 filter_length
 0000000000000000         *UND* 0 filter_cto
 
+RELOCATION RECORDS FOR [PEISDLL1]:
+OFFSET           TYPE              VALUE
+0000000000000008 R_AARCH64_JUMP26  original_entry
+
 RELOCATION RECORDS FOR [PEMAIN01]:
 OFFSET           TYPE              VALUE
 0000000000000018 R_AARCH64_PREL32  start_of_compressed
 000000000000002c R_AARCH64_ABS32   start_of_uncompressed
 
+RELOCATION RECORDS FOR [PETLSHAK]:
+OFFSET           TYPE              VALUE
+0000000000000010 R_AARCH64_ABS32   tls_address
+0000000000000030 R_AARCH64_ABS32   tls_value
+
 RELOCATION RECORDS FOR [PEMAIN02]:
 OFFSET           TYPE              VALUE
 0000000000000014 R_AARCH64_ABS32   comp_len
